src/Core/Content/ImportExport/DataAbstractionLayer/Serializer/Entity/MediaSerializer.php

```diff
@@ -63,6 +63,8 @@ public function deserialize(Config $config, EntityDefinition $definition, $entit
             return $deserialized;
         }
 
+        $url = str_replace(' ', '%20', $url);
```

Comment:
the PHP's built-in urlencode() or rawurlencode() functions did not work? If they would work, I think I would use one of them....

Reply:
Yeah, the `filter_var($url, \FILTER_VALIDATE_URL)` function does not recognize the fully encoded URL as valid. So my approach would be to only encode the whitespaces "manually" with the str_replace.

Reply:
OK ...